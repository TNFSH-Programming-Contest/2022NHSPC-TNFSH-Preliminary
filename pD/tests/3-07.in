6 7
1 3 1 1 2 1
1 2 1 1 2 1 1
.....TT
.T.....
T......
..TTT..
......T
T......
