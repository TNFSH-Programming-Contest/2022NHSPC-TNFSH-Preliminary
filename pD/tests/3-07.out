....ATT
ATA...A
T...A..
.ATTT..
...A.AT
TA.....
